net/upnp/src/opnsense/mvc/app/models/OPNsense/UPnP/ACL/ACL.xml

```diff
@@ -1,14 +1,14 @@
 <acl>
-    <page-service-upnp>
-        <name>Service: Universal Plug and Play</name>
+    <page-services-upnp>
```

Comment:
renaming these is futile and just breaks user setups. You can change the name but the key shouldn't change.